Answers.md

```diff
@@ -1,11 +1,17 @@
 # Answers
 
 1.  What is React JS and what problems does it try and solve?
+    A - React Js is a library that assist in creating UI components. It's great because it solves having to re-render a whole page when state (data) changes. It only renders the component with the changed data.
 
 1.  What does it mean to _think_ in react?
+    A - It means to think of a web-app as a series of smaller components bundled together to make a full web page. Then we have to think what of our state in these components that need to change and how we move the state between different components.
 
 1.  Briefly describe some of the differences between a Class/Stateful component and a Functional/Presentational component.
+    A - A class compoenent carries a state, while a functional component does not.
+        A class component provides multiple methods to help us manage our state and components, while functional components do not.

```

Comment:
Yup pretty much ✅

And remember, class components should hold all the logic where you are manipulating data, functional components should not, they should just be displaying information